test/3x/common/test_common.py

```diff
@@ -0,0 +1,142 @@
+"""Tests for common components.
+Owner(s): ["module: common | auto-tune"]
```

Comment:
this line should be removed?  why mean owner here?

Reply:
Yes, I will remove it later.

Reply:
Removed it in [9b8ec1e](https://github.com/intel/neural-compressor/pull/1575/commits/9b8ec1eb50cf20028985ca30983e42aa528b9b17).